app/src/pages/admin/serverSettingsPage/serverSettingsTabs/statisticsTab/statisticsTab.jsx

```diff
@@ -0,0 +1,212 @@
+  },
+});
+
+@reduxForm({
+  form: 'statisticsTabForm',
+  initialValues: DEFAULT_FORM_CONFIG,
+})
+@connect(null, {
+  showNotification,
+})
+@injectIntl
+export class StatisticsTab extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    initialize: PropTypes.func,
+    handleSubmit: PropTypes.func,
+    showNotification: PropTypes.func,
+  };
+
+  static defaultProps = {
+    initialize: () => {},
+    handleSubmit: () => {},
+    change: () => {},
+    showNotification: () => {},
+  };
+
+  state = {
+    loading: true,
+    listShown: false,
+  };
+
+  componentDidMount() {
+    this.fetchEmailConfig();
+  }
+
+  onFormSubmit = (data) => {
+    this.setState({
+      loading: true,
+    });
+
+    this.updateEmailConfig({
+      method: 'POST',
+      data: {
+        type: STATISTICS_TYPE,
+        enabled: data.enabled,
+      },
+    });
+  };
+
+  onListClick = () => {
+    this.setState((state) => ({
+      listShown: !state.listShown,
+    }));
+  };
+
+  updateEmailConfig = (options) => {
+    fetch(URLS.statisticsServerSettings(), options)
+      .then(() => this.updateSettingSuccess())
+      .catch(this.catchRequestError);
+  };
+
+  fetchEmailConfig = () => {
+    fetch(URLS.serverSettings())
+      .then((data) => {
+        const analyticsResource = data.analyticsResource && data.analyticsResource[STATISTICS_TYPE];
+        const config = {
+          [STATISTICS_ENABLED_KEY]: !!(analyticsResource && analyticsResource.enabled),
+        };
+
+        this.props.initialize(config);
+        this.setState({
+          loading: false,
+        });
+      })
+      .catch(this.catchRequestError);
+  };
+
+  updateSettingSuccess = () => {
+    const {
+      intl: { formatMessage },
+    } = this.props;
+
+    this.props.showNotification({
+      message: formatMessage(messages.updateStatisticsEnabledSuccess),
+      type: NOTIFICATION_TYPES.SUCCESS,
+    });
+
+    this.setState({
+      loading: false,
+    });
+  };
+
+  catchRequestError = (error) => {
+    this.setState({
+      loading: false,
+    });
+
+    this.props.showNotification({
+      message: error.message,
+      type: NOTIFICATION_TYPES.ERROR,
+    });
+  };
+
+  render() {
+    const {
+      intl: { formatMessage },
+      handleSubmit,
+    } = this.props;
+
+    return (
+      <div className={cx('statistics-settings-tab')}>
+        <form className={cx('statistics-settings-form')} onSubmit={handleSubmit(this.onFormSubmit)}>
+          <div className={cx('image-holder')} />
+          <p className={cx('statistics-info')}>{formatMessage(messages.statisticsInfo)}</p>
+
+          <div className={cx('statistics-list', { 'list-shown': this.state.listShown })}>
+            <div className={cx('statistics-list-toggler')} onClick={this.onListClick}>
+              <span>{formatMessage(messages.statisticsList)}</span>
+              <span className={cx('arrow')}>{Parser(ArrowRightIcon)}</span>
```

Comment:
`className={cx('arrow', {rotated: this.state.listShown})}`

Reply:
Done